x/shield/simulation/operations.go

```diff
@@ -150,21 +149,21 @@ func SimulateMsgCreatePool(k keeper.Keeper, ak types.AccountKeeper) simulation.O
 				return simulation.NoOpMsg(types.ModuleName), nil, nil
 			}
 		}
-		shield := sdk.NewCoins(sdk.NewCoin(denom, shieldAmount))
+		shield := sdk.NewCoins(sdk.NewCoin(common.MicroCTKDenom, shieldAmount))
```

Comment:
make sure you don't use common.MicroCTKDenom in any module logics.

Reply:
> make sure you don't use common.MicroCTKDenom in any module logics.

It's not ready for review yet. There're several things to be fixed.